app/controllers/maintaining/iterations_summary_table_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Maintaining::IterationsSummaryTableController < ApplicationController
+  def index
+    @iterations = Iteration.order(created_at: :desc).take(NUMBER_OF_ITERATIONS)
+  end
+
+  private
+  NUMBER_OF_ITERATIONS = 50.freeze
```

Comment:
This is a noop. Integers are singleton instances of the Integer class.

```
50.object_id
50.freeze.object_id
```

It's important for Strings, because strings can be mutated, but ints are immutable so don't need freezing.

Reply:
Okay. Will change